Jenkinsfile

```diff
@@ -91,7 +91,10 @@ node('vetsgov-general-purpose') {
               sh "export IMAGE_TAG=${commonStages.IMAGE_TAG} && docker-compose -p nightwatch-${env.EXECUTOR_NUMBER} up -d && docker-compose -p nightwatch-${env.EXECUTOR_NUMBER} run --rm --entrypoint=npm -e BABEL_ENV=test -e BUILDTYPE=vagovprod vets-website --no-color run nightwatch:docker"
             },     
             cypress: {
-              sh "export IMAGE_TAG=${commonStages.IMAGE_TAG} && docker-compose -p cypress-${env.EXECUTOR_NUMBER} up -d && docker-compose -p cypress-${env.EXECUTOR_NUMBER} run --rm --entrypoint=npm -e CI=true -e NO_COLOR=1 vets-website --no-color run cy:test:docker"
+              sh "export IMAGE_TAG=${commonStages.IMAGE_TAG} && docker-compose -p cypress-${env.EXECUTOR_NUMBER} up -d && docker-compose -p cypress-${env.EXECUTOR_NUMBER} run --rm --entrypoint=npm -e CI=true -e NO_COLOR=1 -e FORMS=false vets-website --no-color run cy:test:docker"
+            },     
+            'cypress-forms': {
+              sh "export IMAGE_TAG=${commonStages.IMAGE_TAG} && docker-compose -p cypressforms up -d && docker-compose -p cypressforms run --rm --entrypoint=npm -e CI=true -e NO_COLOR=1 -e FORMS=true vets-website --no-color run cy:test:docker"
```

Comment:
Recommend adding `-${env.EXECUTOR_NUMBER}` here as well to prevent collisions between docker containers and networks on the same Jenkins node

Reply:
Previous PR for background: https://github.com/department-of-veterans-affairs/vets-website/pull/16837